packages/css-engine/src/core/css-engine.test.ts

```diff
@@ -35,25 +42,57 @@ describe("CssEngine", () => {
       }"
     `);
 
-    engine.addMediaRule(mediaId, mediaRuleOptions0);
+    engine.addMediaRule(mediaId0, mediaRuleOptions0);
     engine.addStyleRule(".c1", {
       style: { color: { type: "keyword", value: "blue" } },
-      breakpoint: mediaId,
+      breakpoint: mediaId0,
     });
     // Default media query should allways be the first to have the lowest source order specificity
     expect(engine.cssText).toMatchInlineSnapshot(`
-      "@media all {
+      "@media all and (min-width: 0px) {
+        .c1 { color: blue }
+      }
+      @media all {
```

Comment:
In queries below `@media all {` is first, comment says that default should be first but here its 2nd

Reply:
huh that's a bug, thank you